app/models/service_template_transformation_plan_task.rb

```diff
@@ -34,9 +34,75 @@ def task_active
     vm_resource.update_attributes(:status => ServiceResource::STATUS_ACTIVE)
   end
 
+  def conversion_host
+    Host.find_by(:id => options[:transformation_host_id])
+  end
+
+  def transformation_log
+    host = conversion_host
+    if host.nil?
+      msg = "Conversion host was not found: ID [#{options[:transformation_host_id]}]. Download of transformation log aborted."
+      _log.error(msg)
+      raise MiqException::Error, msg
+    end
+
+    userid, password = host.auth_user_pwd(:remote)
+    if userid.blank? || password.blank?
+      msg = "Host credential is required: userid [#{userid}], password [#{password}]. Download of transformation log aborted."
```

Comment:
should not log `userid`/`password`. Something like `Credential was not found for host #{host.name}"`.